ovn_k8s/processor/policy_processor.py

```diff
@@ -0,0 +1,67 @@
+# Copyright (C) 2016 Nicira, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at:
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from ovs import vlog
+
+import ovn_k8s.processor as processor
+import ovn_k8s.watcher.policy_watcher
+
+log = vlog.Vlog('policy_processor')
+pw = ovn_k8s.watcher.policy_watcher
+
+
```

Comment:
definitely. I will also add this information to the markdown doc I added in the last commit